app/views/admin_mailer/send_spam_alert.text.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :message do %>
+
+The following accounts have a suspicious level of traffic:<
+ <%@spam_score.sort_by{|k,v| -1*v}.map do |pseud,score| %>
+ <% user = User.find(pseud) %>
+ User <%= user.login%> has a score of <%= score %>
+
+ <%@spam_works[pseud].each { |w| %>
+ <% work = Work.find(w) %>
+  * <%=work.title %> <% if work.akismet_score %>(<%= ts('Flagged as spam') %>)<% end %>
+ <% } %>
+<% end %>
+
+<% end %>
+~              
```

Comment:
Is this `~` here as a divider or something else? If it's intended to be a divider, `<%= text_divider %>` was designed for that.